### ISSUE TITLE
If I download the SDK but dont accept the license, we should show the accept license dialog

### ISSUE THREAD

**Issue by sarahcodes100** [DeerajTheepshi/processing-android#22]
Currently, if I download the SDK but haven't accepted the license, we should show the same "Accept license" dialog when the user tries to run the Sketch and also if they switch out and back into Android

**Comment by kfrajer** [comment]
Maybe a better workflow could be to setup the License from the beginning right before starting the installation? @sarahlensing @Andres @DeerajTheepshi

**Comment by sarahcodes100** [comment]
I don't think this is technically possible, you'd have to run the command to accept the licenses after you download them. Ah, we could also silently accept them automatically unless that violates Processing terms
